build/GridItem.js

```diff
@@ -70,7 +70,7 @@ var GridItem = React.createClass({
     handle: React.PropTypes.string
   },
 
-  getDefaultProps: function () {
+  getDefaultProps: function getDefaultProps() {
```

Comment:
Cool. Thanks a lot for the contribution, I appreciate it.
On 1/8/15 6:50 PM, Jason Brown wrote:
>
> In build/GridItem.js
> <https://github.com/STRML/react-grid-layout/pull/12#discussion-diff-22667656>:
>
> > @@ -70,7 +70,7 @@ var GridItem = React.createClass({
> >      handle: React.PropTypes.string
> >    },
> >  
> > -  getDefaultProps: function () {
> > +  getDefaultProps: function getDefaultProps() {
>
> Will do, I'm at work now so I'll handle all these sometime later tonight.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/STRML/react-grid-layout/pull/12/files#r22667656>.
>